From 8ec6de0e1189174556ddc6bb34b082953bc9f397 Mon Sep 17 00:00:00 2001
From: Alice <alicem@gnome.org>
Date: Fri, 12 Sep 2025 23:35:25 +0400
Subject: [PATCH 16/18] doc: Grammar/typo fixes

Fixes https://gitlab.gnome.org/GNOME/libadwaita/-/issues/1076


(cherry picked from commit c1d3d1436aa93208ee60db0aebc846a4d47b9a14)

Co-authored-by: Alice Mikhaylenko <alicem@gnome.org>

(cherry picked from commit 719db2ed1b6e7219a255b1b609c94cebaa6b2c15)

Co-authored-by: Alice Mikhaylenko <alicem@gnome.org>
---
 doc/css-variables.md | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/css-variables.md b/doc/css-variables.md
index 1b70ea3b..60c0ba8b 100644
--- a/doc/css-variables.md
+++ b/doc/css-variables.md
@@ -69,7 +69,7 @@ are automatically derived from the background color, so it's not necesssary to
 override them manually when setting app-wide accent color.
 
 However, when overriding the background colors for specific widgets, the
-standalone colors must overridden too, as follows:
+standalone colors must be overridden too, as follows:
 
 ```css
 my-widget {
@@ -1202,7 +1202,7 @@ as the following set of variables:
 
 # Compatibility Colors
 
-A number of colors has been available in Adwaita in GTK3. They are aliases of
+A number of colors have been available in Adwaita in GTK3. They are aliases of
 UI colors or otherwise derived from them. These colors use the older
 GTK-specific syntax for named colors rather than CSS variables, and don't pick
 up overridden colors. As such, it's recommended to avoid using these colors
-- 
2.51.1

